nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -98,16 +99,20 @@ def fire_and_forget(
         Returns: None
 
         """
-        if channel == CellChannel.SERVER_COMMAND and topic == ServerCommandNames.HANDLE_DEAD_JOB:
-            if isinstance(targets, list):
-                for target in targets:
-                    self.send_blob(channel=channel, topic=topic, target=target, message=message)
-            else:
-                self.send_blob(channel=channel, topic=topic, target=targets, message=message)
-        else:
-            self.core_cell.fire_and_forget(
-                channel=channel, topic=topic, targets=targets, message=message, optional=optional
-            )
+        # if channel == CellChannel.SERVER_COMMAND and topic == ServerCommandNames.HANDLE_DEAD_JOB:
+        #     if isinstance(targets, list):
+        #         for target in targets:
+        #             self.send_blob(channel=channel, topic=topic, target=target, message=message)
+        #     else:
+        #         self.send_blob(channel=channel, topic=topic, target=targets, message=message)
+        # else:
+        #     self.core_cell.fire_and_forget(
+        #         channel=channel, topic=topic, targets=targets, message=message, optional=optional
+        #     )
```

Comment:
The framework does not need to fobs.dump/fobs.load again since the CoreCell already handle that part.
But why we need to use CoreCell directly in here? this would mean that the streaming/LLM ability is DISABLED?

Reply:
As I mentioned in the description, this is a temporary change. Zhihong will continue to make the changes to integrate with Datum solution.